composer.json

```
@@ -29,6 +29,6 @@
   "scripts": {
     "cs": "./vendor/bin/php-cs-fixer fix --dry-run --diff --config codestyle.php",
     "csf": "./vendor/bin/php-cs-fixer fix --diff --config codestyle.php",
-    "test": "./vendor/bin/phpunit tests --colors always"
+    "test": "./vendor/bin/phpunit tests --colors=always"
   }
-}
+}
```

```suggestion
}

```

Please add missing empty line at the end of the file.